accounts/sombody@somewhere.com.conf.example

```diff
@@ -10,7 +10,6 @@
     	"filter": 50,
     	"hamLearn": 50,
     	"spamLearn": 50
-    },
```

Comment:
This bracket is corresponding to the "batchSize" Object in line 9.
The indentation looks wrong here, no idea why as it is displayed correct in my local IDE.
I will have a look into it.